### PR TITLE
Compartement error msg

### PR DESCRIPTION
**Pull request by ejannett**
Correct error written to the console when compartment is not found

### CODE REVIEW

File: bin/oci-network-inspector [hunk 1]
```diff
@@ -87,10 +87,17 @@ def main():
             comp = sess.find_compartments(args.compartment)
             if comp is not None:
                 comps += comp
+        if len(comps) == 0:
+            __logger.error("Compartement [%s] not found\n" % args.compartment)
```
Comment (wbora) [inline bin/oci-network-inspector]: `Compartement`  should be `Compartment`

File: bin/oci-network-inspector [hunk 2]
```diff
@@ -87,10 +87,17 @@ def main():
...
+            return 1
     else:
         comp = sess.all_compartments()
         comps += comp
 
+    if len(comps) == 0:
+        __logger.error("No compartements found\n")
```
Comment (wbora) [inline bin/oci-network-inspector]: Same comment as above should be `compartment`

File: lib/oci_utils/oci_api.py [hunk 3]
```diff
@@ -1028,31 +1021,17 @@ def get_volume(self, volume_id, refresh=False):
...
-        Parameters
-        ----------
-        compartment_id : str
-            The ID of the wanted compartment.
-        refresh : bool
-            Flag, refresh the information if set.
+    def get_compartment(self, **kargs):
+        if 'ocid' not in kargs:
+            # for now make it mandatory
+            raise StandardError('ocid must be provided')
```
Comment (wbora) [inline lib/oci_utils/oci_api.py]: This is a pretty significant change, have this been tested/validated with all the tools (including kvm)

Comment (ejannett) [reply lib/oci_utils/oci_api.py]: Sorry for confusion. 
Forgot the PR was open for review. I've push this to be able to pull it form another wks 
Testing is still ongoing.

**Comment by ejannett** [comment]
testing/validatin done : 
- align usage of the get_compartment()
- create a virtual network with oci-kvm
- create a Kvm guest on a virtual network
- create a KVM guest on a vmic 
- run oci-network-config --show
- run /usr/bin/oci-network-inspector with band without -C option

**Pull request merged**